evaluator/src/main/kotlin/RuleSet.kt

```diff
@@ -33,7 +35,8 @@ import org.ossreviewtoolkit.utils.ort.log
  */
 class RuleSet(
     val ortResult: OrtResult,
-    val licenseInfoResolver: LicenseInfoResolver = ortResult.createLicenseInfoResolver()
+    val licenseInfoResolver: LicenseInfoResolver = ortResult.createLicenseInfoResolver(),
+    val resolutionProvider: ResolutionProvider = DefaultResolutionProvider.create()
```

Comment:
Does this default make sense? Or should it be `DefaultResolutionProvider.create(ortResult)`?

If the user doesn't provide the `resolutionProvider` should no resolutions be applied at all, or should only the resolutions from the OrtResult be applied?